editor/filesystem_dock.cpp

```diff
@@ -2385,6 +2385,11 @@ void FileSystemDock::_file_option(int p_option, const Vector<String> &p_selected
 			}
 		} break;
 
+		case FILE_FAVORITE_SHOW_IN_FILESYSTEM: {
+			String fpath = p_selected[0];
+			FileSystemDock::get_singleton()->navigate_to_path(fpath);
```

Comment:
I'm seeing other `case`s here being defensive about checking to see that `!p_selected.is_empty()` first. I'm not sure if it's just to be safe or if this list could actually end up being empty, but it's probably best to do the same either way.

You're also already in `FileSystemDock`, so going through the singleton shouldn't be necessary.

```suggestion
			if (!p_selected.is_empty()) {
				navigate_to_path(p_selected[0]);
			}
```

Reply:
done